bottomnavigation/sample/src/main/java/com/microsoft/device/dualscreen/sample/bottomnavigation/MainActivity.kt

```diff
@@ -59,7 +56,7 @@ class MainActivity : AppCompatActivity() {
             allowFlingGesture = true
         }
 
-        initWindowLayoutInfo()
+        registerWindowInfoFlow()
```

Comment:
Why this rename? It doesn't really shows what the functions does, and uses `flow` word that, yeah it's used inside but doesn't really provide any value. Why not something like `setupWindowInfoTracker`? or something simpler? (even the older name could work). My 0.02$

Reply:
I've seen this refactor here and there along the PR. I've seen that it was inconsistency between other libs/samples and the name used. Since you have now made the name cohesive between all modules, we can leave it as it is (with your change.

Reply:
I used the registerWindowInfoFlow because it was more prevalent in the SDK and Bundle app.
Since this is a function name that we will use a lot, we can discuss and choose something else.